packages/samples/vuejs/README.md

```diff
@@ -1,46 +1,157 @@
-# vue-search
+# Vue.js Search Page
```

Comment:
As an intro we could explain the purpose of this project/package, which is to showcase an implementation with vuejs + Atomic, and to work out the quirks of such an implementation.

And this this is not meant to be used as is in production, but as a demonstration of some patterns + configuration.

The goal is that if someone stumble on this README while browsing our repo, they should quickly understand that it's "code sample".